js/build.js

```diff
@@ -34,9 +34,11 @@ Fliplet.Widget.instance('image', function (data) {
   }).on('error', function(){
     $placeholder.fadeInImg(this);
 
-    if (Raven) {
+    if (typeof Raven !== 'undefined' && Raven.captureMessage) {
       Raven.captureMessage('Error loading image', {
         user: Fliplet.User.get('id'),
+        app: Fliplet.Env.get('appId'),
```

Comment:
just naming but why not appId/pageId like we usually set?

Reply:
Dunno. I do sometimes just use `app`.

Reply:
we should definitely have a JS API to handle this...

```
Fliplet.Errors.capture
```